wheel/lib_file_analyse.py

```diff
@@ -0,0 +1,334 @@
+"""
```

Comment:
I'd prefer this module to be named `macos_libfile` unless it's being used for other platforms as well.

Reply:
Done but on `macosx_libfile` because of grammar check.